### ISSUE TITLE
Getting Involved

### ISSUE THREAD

**Issue by RichieBzzzt** [vsoch/pokemon#4]
Hello! 

I forked and made some changes to the make_db scripts and had to do a few other updates as the underlying website had been re-designed so the database could no longer be rebuilt.

I've got the changes here ready to go; you happy for me to raise a PR?

https://github.com/richiebzzzt/pokemon

I've tested that it works on Windows 10, Ubuntu and even a Databricks Cluster! My test package is on pypi (clefairy)

Thanks for creating this, it's great!

**Comment by vsoch** [comment]
Yes a PR would be great! Thanks for fixing things up, I haven’t looked at this guy in a while (but have used many times over the years).
